coral-component-popover/src/templates/base.html

```diff
@@ -1,3 +1,3 @@
-<div class="_coral-Dialog-header" handle="headerWrapper"></div>
-<button tracking="off" handle="closeButton" class="_coral-Dialog-closeButton _coral-ClearButton" type="button" is="coral-button" variant="_custom" icon="close" title="{{data.i18n.get('Close')}}" tabindex="-1" coral-close></button>
-<div class="_coral-Popover-tip" handle="tip"></div>
+<div class="_coral-Dialog-header" handle="headerWrapper" role="presentation"></div>
+<button tracking="off" handle="closeButton" class="_coral-Dialog-closeButton _coral-ClearButton" type="button" is="coral-button" variant="_custom" icon="close" aria-label="{{data.i18n.get('Close')}}" title="{{data.i18n.get('Close')}}" tabindex="-1" coral-close></button>
+<div class="_coral-Popover-tip" handle="tip" aria-hidden="true"></div>
```

Comment:
So if you just use `hidden` attribute, screen reader will still read ? You have to specify `aria-hidden` ?